openwisp_users/admin.py

```diff
@@ -67,8 +67,7 @@ def get_formset(self, request, obj=None, **kwargs):
         formset = super(OrganizationUserInline, self).get_formset(request, obj=obj, **kwargs)
         if not request.user.is_superuser:
             operator_orgs = OrganizationUser.objects.values_list('organization') \
-                                                    .filter(user=request.user,
-                                                            is_admin=True)
+                                                    .filter(user=request.user)
```

Comment:
The problem with removing `is_admin=True` here is that if a user say, 
A is an operator who has the permission to view and edit users of his organization at the same A is also a user of another organization with no `is_admin` permission in the other organization, he'll be able to edit the data of the other organization.

To understand and realize this problem, try this:
1. Make two orgs: `org1` and `org2`
2. Make a user `testuser` who is an operator and has organization user relation with both the orgs but `is_admin` in org1 and not in org2.
3. Go to the `change/` page of this user after logging in to the operator account.

Notice: here you can make this user as `is_admin` for org2 as well or delete `org2` relationship, which should not be allowed! 

Please let me know if you understand what i am trying to say here,

If i am clear till this point then:
I think the ideal solution for this problem would be to hide the information about the org that the user has no access to edit, what do you think? :smile: 

P.S: I am also interested to know if you've found the discussion on operator and have properly made the operator user in your system's database, if there is any doubt, please feel free to ping here about that as well. :smile:

Reply:
Yes you are right @atb00ker. i created the `testuser `as instructed and when login in, this user was able to change and edit `org2` which is not suppose to happen. I think the solution you proposed is ideal. i will looked into how to implement it. 
Thanks.